translations/fr.po

```diff
@@ -6,7 +6,7 @@
 msgid ""
```

Comment:
Hmm, what is this merge commit doing here?

Reply:
@yurug I don't see anything wrong here: these lines are already there in master:

https://github.com/ocaml-sf/learn-ocaml/blob/master/translations/fr.po#L6-L7

and given this is `""`, not `"\n"`, these two lines are basically some no-op.

Reply:
Regarding the `merge commit`: it's on-purpose.

A rebase or a merge was necessary to fix the merge conflict.

However, given the PR will be merged in a squash strategy, it appears to me it's OK and even simpler to do a merge from master to the PR branch, rather than a rebase (a rebase would not preserve @Plictox' local history, and may lead to more conflicts).